Initial commit of Rapido

AaronHerrick opened the pull request:
Badnoobs Rapido

AaronHerrick committed:
Initial commit of Rapido

Badnoobs Rapido

KiloQubit reviewed (changes requested):
All the other toolheads are directly in the `Toolheads` directory with just filenames differentiating them.  For consistency, please either put your toolhead directly in `Toolheads`, or create directories for all the other toolheads in there and move them as appropriate.  If you decide to have separate directories for each toolhead, please make sure there are no spaces in their names, like `Phaetus Rapido`

AaronHerrick committed:
reorganizing toolheads

by using separate directories for each toolhead its much easier to find the toolhead files.

AaronHerrick committed:
fixing orientation of stls

I had the stls turned a strange way

AaronHerrick committed:
moved text to the top of the toolhead

AaronHerrick committed:
fixing stl issues caused by moving text

AaronHerrick commented:
the files here are obsolete
